src/core.js

```diff
@@ -729,17 +731,45 @@ define([
             tableData = form.findUsages();
 
         $modal.addClass('fd-full-screen-modal');
+        $modalBody.append($('<div>').append($('<label for="find-usages-search">').text("Search")));
+        $modalBody.append($('<div>').append($('<input id="find-usages-search">').attr('type', 'text')));
```

Comment:
Can the search box be styled more like the data browser search box, with a magnifying glass icon to the left of the search box rather than a *Search* label above it?